README.md

```diff
@@ -1,70 +1,65 @@
-# Getting Started with Create React App
+<p align="center">
+  <a href="https://github.com/ixboy/countries-covid19-tracker">
+    <img alt="GitHub last commit (branch)" src="https://img.shields.io/github/last-commit/ixboy/countries-covid19-tracker/main?color=blue&style=flat-square">
+  </a>
+  <a href="https://github.com/ixboy/countries-covid19-tracker">
+    <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/ixboy/countries-covid19-tracker?color=pink&label=%E2%98%85%20stars%20&style=flat-square">
+  </a>
+  <a href="https://github.com/ixboy">
+    <img alt="GitHub followers" src="https://img.shields.io/github/followers/ixboy?color=yellow&logo=github&style=flat-square">
+  </a>
+</p>
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Covid-19 World Data Tracker
```

Comment:
- [ ] In keeping with the Creative Commons License, in your README kindly add acknowledgment for the design author, Nelson Sakwa.

Reply:
@TedLivist  Thank You for your reviews I really appreciate. However I am pretty sure my design is way too different from Nelson Sakwa's designe, I don't see why I should give him credits if I did not copy from his design.
Anyways I just gave him credits just for the sake of getting my project approved...